deepchecks/utils/metrics.py

```diff
@@ -157,7 +157,9 @@ def validate_fitting(self, model, dataset: 'base.Dataset', should_return_array:
                                                   f'returned only {len(result)} elements in the score array value')
 
         else:
-            result = self._run_score(model, df.head(2), dataset)
+            # In order for scorer to return array in right length need to pass him samples from all labels
+            single_label_data = df.groupby(dataset.label_name).head(1)
```

Comment:
in here the scorer should return number so no need for the group by (and also comment is incorrect)

Reply:
The groupby fixes the bug - in order to run, the sklearn scorer must have at least one sample of each label

About the comment-  will fix